src/components/RangeSlider/components/DualThumb/DualThumb.scss

```diff
@@ -98,9 +98,6 @@ $range-thumb-border-error: rem(2px) solid color('red');
     @include focus-ring($style: 'focused');
     outline: 0;
     border-color: var(--p-override-transparent);
-    @media (-ms-high-contrast: active) {
-      outline: 1px solid ms-high-contrast-color('text');
-    }
```

Comment:
Does this need the high contrast mixin? I think the `border-color: var(--p-override-transparent);` should be removed

Reply:
I removed the high-contrast mixin, and you're right, removing border-colour!